src/main/java/seedu/duke/logger/LoggerUtil.java

```diff
@@ -11,15 +12,16 @@
 public class LoggerUtil {
     public static ConsoleHandler ch;
     public static FileHandler fh;
+    private static final String LOGGER_PATH = "log/programLog.log";
+    private static final File FILE = new File(LOGGER_PATH);
     private static final Logger LOGGER = Logger.getLogger(LoggerUtil.class.getName());
 
     public static void loggerUtilSetup() {
         LogManager.getLogManager().reset();
-        ch = new ConsoleHandler();
-        ch.setLevel(Level.SEVERE);
-        LOGGER.addHandler(ch);
```

Comment:
Maybe you could let this code stay cause in the case the filehandler is not created and the exception is caught then the console handler needs to show the log
Removing it from the `setupLogger()` function will still ensure that none of the other classes log into the console, only if there's an exception in `loggerUtilSetup()` then it is logged